testsuite/tests/kuadrant/conftest.py

```diff
@@ -24,15 +40,15 @@ def authorization_name(blame):
 
 
 @pytest.fixture(scope="module")
-def authorization(kuadrant, oidc_provider, route, authorization_name, openshift, label):
+def authorization(kuadrant, route, authorization_name, openshift, label):
     """Authorization object (In case of Kuadrant AuthPolicy)"""
     if kuadrant:
         return AuthPolicy.create_instance(openshift, authorization_name, route, labels={"testRun": label})
     return None
 
 
 @pytest.fixture(scope="module")
-def rate_limit(kuadrant, openshift, blame, request, module_label, route, gateway):
```

Comment:
do all other tests pass? Not sure if this is the case for these specifically, but we sometimes do unused imports to make sure everything is ready beforehand